packages/payment-detection/src/any/retrievers/thegraph.ts

```diff
@@ -1,13 +1,14 @@
-import { PaymentTypes } from '@requestnetwork/types';
 import { CurrencyDefinition } from '@requestnetwork/currency';
+import { PaymentTypes } from '@requestnetwork/types';
 import { BigNumber, utils } from 'ethers';
+import { IPaymentRetriever } from 'payment-detection/src/types';
```

Comment:
I guess this is VS Code doing, but this kind of import won't work in JS. I think this works because it's only a type, but I recommend against it


```suggestion
import { IPaymentRetriever } from '../types';
```